src/css/tailwind.css

```diff
@@ -235,6 +235,19 @@
   color: #d4d4d4 !important;
 }
 
+.menu_node_modules-\@docusaurus-theme-classic-lib-theme-DocSidebar-Desktop-Content-styles-module {
```

Comment:
This selector will not work on the built project because it removes everything after the menu and adds a different generated string.